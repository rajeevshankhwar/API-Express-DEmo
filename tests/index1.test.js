const index = require("../index1");
test('first Test', () => {

    const result = index.get('/status', (req, res) => {
        console.log("Looks like server is running ok");
        res.status(200).send("Looks like everything on API server is working");
        expect(result).toBe(1);
    });
});