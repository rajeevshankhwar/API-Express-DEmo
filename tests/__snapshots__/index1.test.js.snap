// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`first Test 1`] = `[TypeError: index.get is not a function]`;
